test/erlzk_test.erl

```diff
@@ -31,7 +31,9 @@ erlzk_test_() ->
 
 setup() ->
     erlzk:start(),
-    {ok, [ServerList, Timeout, Chroot, AuthData]} = file:consult("../test/erlzk.conf"),
+    Dir = code:priv_dir(erlzk),
+    ConfFile = filename:join([Dir, "erlzk.conf"]),
```

Comment:
I think you forgot to add `priv/erlzk.conf` to the repo.
On the other hand, the file could stay where it is (and where it belongs, imho), just look up the path with `code:lib_dir(erlzk, tests)`.